lib/features/evaluate/config.dart

```diff
@@ -407,7 +457,7 @@ class EvaluateConfigState extends ConsumerState<EvaluateConfig> {
           spacing: configWidgetSpace,
           children: [
             configLeftGap,
-            Text('Eavluation Dataset: '),
+            Text('Evaluation Dataset: '),
```

Comment:
Conditionalise 'Evaluation' to be either 'Tuning' or 'Evaluation' depending on the settings provider.

Reply:
Wrong bit of code - all good.